PanKanapka/tools/views.py

```diff
@@ -0,0 +1,16 @@
+# Create your views here.
+from django.shortcuts import render
+from django.http import HttpResponse
+from .csv2db import DbManager
+
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the database tools index.")
+
+
+def fill_databases(request):
+    return HttpResponse("Użyj >>python manage.py filldb<< z linii poleceń")
```

Comment:
Myślę, że jeśli ta funkcjonalność nie jest wywoływana urlem, to można spokojnie wyrzucić te metody.